src/both/DeltaNeutralLpV3.sol

```diff
@@ -163,6 +173,9 @@ contract DeltaNeutralLpV3 is AccessStrategy {
     /// @notice Gives ratio of vault asset to borrow asset, e.g. WMATIC/USD (assuming usdc = usd)
     AggregatorV3Interface immutable borrowAssetFeed;
 
+    address public constant WETH = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    uint256 public immutable decimalDiff;
```

Comment:
I think we can use a function like [`_giveDecimals`](https://github.com/AffineLabs/contracts/blob/4c00d3f672e748c4eb39083e4c794feb000e55b8/src/ethereum/ConvexStrategy.sol#LL174C5-L182C6) to handle decimal conversions.

Reply:
The new version is cleaner.